modules/oe_translation_poetry/tests/Functional/PoetryTranslationRequestTest.php

```diff
@@ -275,6 +287,302 @@ public function testUnprocessedJobHandling(): void {
     $this->assertJobsPoetryRequestIdValues($this->jobStorage->loadMultiple(), $expected_poetry_request_id);
   }
 
+  /**
+   * Tests the handling of Poetry status notifications.
+   */
+  public function testStatusNotifications(): void {
```

Comment:
I would put this and next tests in a different place e.g. `PoetryTranslationNotificationTest`.

Reply:
Yes, true, will do.